cosmetics-web/config/initializers/sidekiq.rb

```diff
@@ -11,9 +11,23 @@ def create_log_db_metrics_job
   end
 end
 
+def create_elasticsearch_index_job
+  job = Sidekiq::Cron::Job.new(
+    name: "Reindex Elasticsearch, every day at 1 am",
+    cron: "* 1 * * *",
+    class: "ReindexElasticsearchJob",
+    queue: "cosmetics",
+  )
+  unless job.save
+    Rails.logger.error "***** WARNING - Elasticsearch reindexing job was not saved! *****"
+    Rails.logger.error elasticsearch_job.errors.join("; ")
```

Comment:
From the diff seems `elasticsearch_job` is not defined and we should use `job` instead:
```suggestion
    Rails.logger.error job.errors.join("; ")
```